tcf_website/views/profile.py

```diff
@@ -3,10 +3,12 @@
 from django.shortcuts import render
 from django.contrib.auth.decorators import login_required
 from django.contrib import messages
+from django.db.models import Avg, Count, Q
 from django import forms
 from django.forms import ModelForm
 from django.http import HttpResponseRedirect
-from ..models import User
+from .browse import safe_round
```

Comment:
Little bit unsure if it's a good idea to import helper functions from another view, but I can't think of a better way to do it — only alternatives would either be duplicating the function or creating some new file they can both import from, which don't strike me as particularly good options.

Reply:
Do you think it's better to create `utils.py`, which keeps all the helper functions?

Reply:
> Do you think it's better to create `utils.py`, which keeps all the helper functions?

That sounds like a good idea to me

Reply:
Let's discuss this in https://github.com/thecourseforum/theCourseForum2/issues/303 since it also has to do with `emdashOrTwoDecimals` in `tcf_website/static/department/loadMoreCourses.js`. Marking the conversation as resolved.